src/domain/human-description/entities/human-description.entity.ts

```diff
@@ -1,37 +1,44 @@
 import { Hex } from 'viem/src/types/misc';
 
 export enum ValueType {
-  Word = 'word',
+  Text = 'text',
```

Comment:
Afaik it should be changed everywhere now. Can you point me to where it needs to be updated still?

Reply:
Ah thanks for the explanation. Now I understand. I agree in this case it works but for other value types it would also require changes to the JSON which we don't want to do. Also good find with the regex. Lets rename it there to `textToken` as well.